CMakeLists.txt

```diff
@@ -413,6 +413,17 @@ option(CAPNP_DISABLE "Disable Cap'nProto (needed for cosimulation).")
 if (CAPNP_DISABLE)
   message (STATUS "Disabling Cap'nProto.")
 else()
+  # Some of our CMake code which is only used in Capnp builds uses newer CMake
+  # features. Since most compilations are not capnp builds, only require the new
+  # version for those builds.
+  if(CMAKE_VERSION VERSION_LESS "3.23.0")
+    message(SEND_ERROR
+        "Building CIRCT with Cap'nProto features enabled is CMake features "
```

Comment:
`is CMake features` seems off... `uses CMake features`?

Reply:
I just copy-pasted from the python stuff and didn't copy edit. Oops!